src/routes/(site)/about/+page.svelte

```diff
@@ -4,7 +4,6 @@
 	import { fly } from 'svelte/transition';
 	import emo from '$assets/emo.jpg';
 	import bboy from '$assets/bboy.jpg';
-	import benjamin from '$assets/benjamin.jpg';
```

Comment:
gotta remove this image from the codebase too

Reply:
dope thanks for this. going to give workspace another run and will see if we can use your review to delete.